TF_Application/Assets/TouchFree_Application/Scripts/TouchFreeMain.cs

```diff
@@ -48,5 +58,39 @@ public static void UpdateCursorWindowSize(float _cursorSize)
             CursorWindowSize = newWindowSize;
             CursorWindowMiddle = Vector2.one * (CursorWindowSize / 2);
         }
+
+        IEnumerator CheckForSettingsOpen()
+        {
+            WaitForSeconds wait = new WaitForSeconds(1);
+
+            yield return wait;
+
+            while (true)
+            {
+                IntPtr hwnd = FindWindow(null, "TouchFreeService");
+
+                if ((int)hwnd != 0)
+                {
+                    if (!settingsOpen)
+                    {
+                        disableWhenSettingsOpen.SetActive(false);
```

Comment:
This would be better handled with a method that makes it clear what objects we are disabling/enabling.  At the moment I would need to open the scene to find that out, but ideally we should be able to follow the flow of the code without opening the scene.